charts/incubator/drone-runner/questions.yaml

```diff
@@ -0,0 +1,167 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{controller}
+# Include{controllerDeployment}
+# Include{replicas}
+# Include{replica1}
+# Include{strategy}
+# Include{recreate}
+# Include{controllerExpert}
+# Include{controllerExpertExtraArgs}
+  - variable: env
+    group: "Container Configuration"
+    label: "Image Environment"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: DRONE_RPC_HOST
+          label: "DRONE_RPC_HOST"
+          description: "DRONE RPC HOST"
+          schema:
+            type: string
+            default: ""
+            required: true
+        - variable: DRONE_RPC_PROTO
+          label: "DRONE_RPC_PROTO"
+          description: "DRONE RPC PROTO"
+          schema:
+            type: string
+            default: "http"
+            required: true
```

Comment:
This can be an enum. since it's only possible for 2 values

Reply:
Or remove this completely from GUI and set it on values.yaml only like that:
`{{ .Values.service.main.ports.main.protocol | lower }}`